Troubleshooting.md

```diff
@@ -0,0 +1,33 @@
+# Troubleshooting
+If you have problems with the La Capitaine Icon Theme, please read the following sections before submitting an issue on GitHub. `.desktop` files usually can be found in `/usr/share/applications/` (accessible for all users) or in `~/.local/share/applications/` (accessible for the current user). To see if some application icon is available in La Capitaine, just browse `apps/scalable/` in this repository and have a look. If you got lucky, the icon can be applied to any application by placing `Icon=cool-icon-name` in the application's `.desktop` file, assuming `apps/scalable/cool-icon-name.svg` exists.
+
+## No icon gets themed
+### Your environment is not supported
+This could be due to your desktop environment or system not being supported. Some older environments, mostly ugly ones :wink:, are not supporting SVG icons. The solution is rasterizing the vector graphics, which is not planned for now. See [the Wiki section for supported environments](https://github.com/keeferrourke/la-capitaine-icon-theme/wiki#supported-environments) for more information.
```

Comment:
I would remove the comment about ugly desktops. Everyone has different ideas of aesthetic, and it's unfair to cast windowmaker and dwm aside as ugly. #allDesktopsMatter :wink: